src/lambda_vm/interpreter.mli

```diff
@@ -13,4 +13,11 @@ type script_result = {
   storage : Ir.value;
   operations : unit;
 }
-val execute : Gas.t -> arg:Ir.value -> Ir.code -> (script_result, error) result
+
+type context = {
```

Comment:
please move it to a separate module and create a ```make``` function for building the context and functions to interact with it through abstract module